kotlin/kotlin/pom.xml

```diff
@@ -45,7 +45,6 @@
 					<verbose>false</verbose>
 					<showTree>false</showTree>
 					<grammarName>Kotlin</grammarName>
-					<caseInsensitive>true</caseInsensitive>
```

Comment:
This element is illegal. "true" is not a valid value for caseInsensitive. The Maven plugin doesn't check for bad values, but trgen now does, forcing the user to correct bad information before it gets copied and pasted around into other grammars by people who don't know what they are doing.

Reply:
For Antlr 4.10, it seems that "caseInsensitive=true" is only available as an option in the grammar options section. _It isn't available as a command-line option on the call for the tool (nowhere [here](https://github.com/antlr/antlr4/blob/1b144fa7b40f6d1177c9e4f400a6a04f4103d02e/tool/src/org/antlr/v4/Tool.java#L121))_. Therefore, "caseInsensitive=true" should be removed from the pom.xml file, and the rest of the occurrences of caseInsensitive will need to be changed. https://github.com/antlr/antlr4/issues/3436 https://github.com/antlr/antlr4/commit/d8e152592fb4dbdd6c9fda2c169031cca17dfc4f